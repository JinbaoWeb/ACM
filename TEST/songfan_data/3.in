56 45 11
1 4198
2 7795
3 9484
4 9650
5 4590
6 6431
7 705
8 8316
9 5557
10 8189
11 2652
12 606
13 2153
14 7829
15 9813
16 367
17 6658
18 8961
19 1039
20 85
21 8917
22 7167
23 4895
24 3440
25 5962
26 2424
27 9711
28 7512
29 8448
30 95
31 8702
32 4489
33 9599
34 8588
35 7723
36 7212
37 1614
38 8921
39 7874
40 7976
41 1654
42 12
43 3749
44 1512
45 8071
5 3
9 2
10 3
23 14
25 0
26 1
27 2
28 0
30 4
34 2
34 3
