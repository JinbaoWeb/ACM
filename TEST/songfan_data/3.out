15 27 4
33 3
18 38 21
31 34 29 8 10 45 40 39 14 2 35 28 36 22

17
6 25

9 23 5 32
1 43
24 11 26
